alacritty_terminal/src/clipboard.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 Joe Wilm, The Alacritty Project Contributors
+// Copyright 2020 Christian Duerr, The Alacritty Project Contributors
```

Comment:
Did you change this to Christian Duerr because jwilm never contributed anything to this file?

Honestly I'd just either remove this header or change it to `Copyright 2020 The Alacritty Project Contributors`/`Copyright 2020 The Alacritty Project`. That way it's at least universal and one wouldn't have to worry about which name it should be.